include/IECoreScene/MeshAlgo.h

```diff
@@ -53,7 +53,21 @@ namespace MeshAlgo
 /// If provided, this will be periodically checked, and cancel the calculation with an exception
 /// if the canceller has been triggered ( indicating the result is no longer needed )
 
+// Enum for specifying how to weight normal calculation
+enum class NormalWeighting
+{
+	EqualVertex, // Equal weight to each vertex, simple, not very good
+	VertexAngle, // Weight based on angle where each face touches vertex - best consistent results
```

Comment:
This also sounds slightly odd, I think because a Vertex doesn't have an angle. Is CornerAngle any better?

Reply:
I guess I was just matching Houdini, which calls this "By Vertex Angle". It's not like this terminology is uncommon ( i.e. https://mathworld.wolfram.com/VertexAngle.html ), but I can see the argument that we use Vertex to refer to vertices of the mesh, not of polygonal faces. Does FaceVertexAngle sound better to you? I don't think I feel strongly if you prefer CornerAngle.

Reply:
I wondered about FaceVertexAngle, but it seemed kinda wordy. Maybe just `Angle`? `Uniform`, `Angle` and `Area`? I don't mind too much really, although I'll probably get picky about how we present it to the user in Gaffer.

Reply:
I've gone with Equal, Angle, and Area.